src/components/navigation/SearchInput.tsx

```diff
@@ -153,20 +157,29 @@ const _Input = styled.input<{ focused: boolean }>`
   border: 0;
   outline: none;
 
-  margin-left: 4px;
   cursor: ${props => (props.focused ? 'auto' : 'pointer')};
 
   &::placeholder {
     color: ${props => props.theme.colors.brand};
   }
 
+  ::-webkit-input-placeholder {
+    opacity: 1;
+  }
+  ::-moz-placeholder {
+    opacity: 1;
+  }
+  :-ms-input-placeholder {
+    opacity: 1;
+  }
+
```

Comment:
you can add `opacity: 1` to the rule in 162-164 and remove the vendor-prefixes. I just checked it, styled components are generating them automatically

Reply:
nice!